lib/src/pull_down_button.dart

```diff
@@ -336,6 +337,11 @@ class PullDownButton extends StatefulWidget {
   @experimental
   final Alignment? animationAlignmentOverride;
 
+  /// Custom navigator used to show the pull-down menu.
```

Comment:
Could you also add what purpose this property has? Maybe mention that nested navigator use case?